packages/firestore/src/api/aggregate.ts

```diff
@@ -14,25 +14,52 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import { firestoreClientRunAggregationQuery } from '../core/firestore_client';
-import { AggregateQuery, AggregateQuerySnapshot } from '../lite-api/aggregate';
+import { Query, SnapshotOptions } from '../api';
+import { firestoreClientRunCountQuery } from '../core/firestore_client';
+import {
+  AggregateField,
+  AggregateQuerySnapshot as AggregateQuerySnapshotLite,
+  AggregateSpec,
+  AggregateSpecData
+} from '../lite-api/aggregate';
 import { cast } from '../util/input_validation';
 
 import { ensureFirestoreConfigured, Firestore } from './database';
 
 export {
-  AggregateQuery,
+  AggregateField,
+  AggregateSpec,
+  AggregateSpecData,
   AggregateQuerySnapshot,
-  aggregateQueryEqual,
-  aggregateQuerySnapshotEqual,
-  countQuery
+  aggregateSnapshotEqual,
+  aggregateFieldEqual
 } from '../lite-api/aggregate';
 
-export function getAggregateFromServerDirect(
-  query: AggregateQuery
-): Promise<AggregateQuerySnapshot> {
-  const firestore = cast(query.query.firestore, Firestore);
+/**
```

Comment:
should I remove this, and export lite AggregateQuerySnapshot directly? Or should I add the metadata with fixed false values in this extended AggregateQuerySnapshot and export it?